CMakeLists.txt

```diff
@@ -46,6 +46,9 @@ if(BUILD_TESTING)
     add_subdirectory(tests)
 endif()
 
+# Leveldb library
+add_subdirectory(wrappers/leveldb)
```

Comment:
You should add a CMake command-line option to make building the LevelDB wrapper optional. This option should be OFF by default. You can declare the option in the beggining of the file with:
```cmake
option(PIBENCH_BUILD_LEVELDB "Build LevelDB wrapper" OFF
```

And enclose the `add_subdirectory(wrappers/leveldb)` with:
```cmake
if(PIBENCH_BUILD_LEVELDB)
    add_subdirectory(wrappers/leveldb)
endif(PIBENCH_BUILD_LEVELDB)
```